src/index.ts

```diff
@@ -149,6 +153,10 @@ const typescript: PluginImpl<RPT2Options> = (options) =>
 			service = tsModule.createLanguageService(servicesHost, documentRegistry);
 			servicesHost.setLanguageService(service);
 
+			if(!pluginOptions.clean && shouldEnableDefaultClean) {
+				context.warn("You have enabled transformers or sourceMapCallback, but have disabled the default clean option. You may need to use 'extraCacheKeys' option to enable it again.")
```

Comment:
```suggestion
				context.warn("You have enabled 'transformers' or 'sourceMapCallback'. To enable caching, you will have to configure the 'extraCacheKeys' option. See https://github.com/ezolenko/rollup-plugin-typescript2#plugin-options")
```

what the default is or isn't doesn't need to be specified. Just that caching is disabled. This is user-facing, so we want to make the message as simple and to-the-point as possible.

_Ideally_ we should also say _which_ of the two (or both) are enabled, as that makes for a better/more informative warning.